roles/os_net_setup/defaults/main.yml

```diff
@@ -11,8 +11,8 @@ cifmw_os_net_setup_config:
     subnets:
       - name: public_subnet
         cidr: 192.168.122.0/24
```

Comment:
so that subnet is the same as the osp_trunk we have, which collide with the `default` network in libvirt.

Would it be possible to get a parameter for that subnet (and use ansible.builtin.ipmath filter to calculate IPs based on that parameter subnet) ? That way, we may be able to consume what networking-environment-definition provides instead of that hard coded subnet, without having to override the whole dict. WDYT?

@pablintino any thoughts? Am I completely out of topic?

Reply:
@cjeanner It has nothing to do with that. Here we create a new neutron network leveraging the unused  very end of the controlplane network, that lasts till the .200 IP, included. As @karelyatin says, that IP is used by the control-plane, so it shouldn't be used by the new OSP network.

Reply:
Yes right if the underlying network is change it would have an impact here. So would need a way to sync these.